sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -501,7 +538,7 @@ private synchronized void cancelPreviousTasks() {
      * Method that triggers the distribute workflow or proceed to the next step.
      */
     private synchronized void resumeDistributeWorkflow() {
-        if (mPackageInfo != null && mForegroundActivity != null && !mWorkflowCompleted && isInstanceEnabled()) {
+        if (mPackageInfo != null && mForegroundActivity != null && !mWorkflowCompleted) {
```

Comment:
Oh wait you are right, there are 2 posts here, state could change again.
I actually tested that on push. I am just missing a test on distribute.